README.md

```diff
@@ -565,7 +565,7 @@ Flexbox is all about having a parent element with `display: flex;` that controls
 }
 ```
 
-See [the flexbox example page](https://davidhartsough.com/you-dont-need-bootstrap/flexbox.html) and [the source code](https://github.com/you-dont-need-bootstrap/blob/master/flexbox.html).
+See [the flexbox example page](https://davidhartsough.com/davidhartsough/you-dont-need-bootstrap/flexbox.html) and [the source code](https://github.com/davidhartsough/you-dont-need-bootstrap/blob/master/flexbox.html).
```

Comment:
The example page url needs to be changed to `https://davidhartsough.com/you-dont-need-bootstrap/flexbox.html` @davidhartsough

Reply:
Nice! Thank you for catching this! I'll fix it now :)